test/checks/dangerousCapabilities/failure.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+  labels:
+    app.kubernetes.io/name: nginx
+spec:
+  containers:
+  - name: nginx 
+    image: nginx
+    securityContext:
+      capabilities:
+        add:
+        - NET_ADMIN
```

Comment:
Can we add a similar `failure.all.yaml` which adds `ALL` instead of `NET_ADMIN`?